<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CliParserDialog</class>
 <widget class="QDialog" name="CliParserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>941</width>
     <height>511</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Segment mode:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButtonConvert">
      <property name="text">
       <string>convert</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0" colspan="5">
     <layout class="QHBoxLayout" name="layoutSplitter"/>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBoxSeg">
      <item>
       <property name="text">
        <string>auto</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0.5 - 0.5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1 - 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1 - 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2 - 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2 - 2</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QCheckBox" name="checkBoxRfc">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>rfc</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>7</y>
     <width>941</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layoutToolbar"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
